cves/CVE-2008-0005.yml

```diff
@@ -217,4 +221,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Very interesting read on an additional way to commit XSS again. Very clever. +4 Upvotes